### PR TITLE
[fix](mtmv)fix can not create mtmv all use default value

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 1]
```diff
@@ -616,6 +616,9 @@ private String getOriginSql(ParserRuleContext ctx) {
 
     @Override
     public MTMVRefreshTriggerInfo visitRefreshTrigger(RefreshTriggerContext ctx) {
+        if (ctx == null) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:619]: If all are default value, better create a default RefreshTriggerContext?

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:619]: I got it

**Review by morningman — APPROVED** [review]
LGTM

**Review by seawinde — APPROVED** [review]

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**